File: Java_Course/OOP4/src/com/sharan/Monitor.java
package com.sharan;

public class Monitor {
    private String model;
    private String manufacterer;
    private int size;
    private Resolution nativeResolution;

    public Monitor(String model, String manufacterer, int size, Resolution nativeResolution) {
        this.model = model;
        this.manufacterer = manufacterer;
        this.size = size;
        this.nativeResolution = nativeResolution;
    }

    public String getModel() {
        return model;
    }

    public String getManufacterer() {
        return manufacterer;
    }

    public int getSize() {
        return size;
    }

    public Resolution getNativeResolution() {
        return nativeResolution;
    }

    public void drawPixelAt(int x, int y, String color){
        System.out.println("Drawing pixel at ("+x+","+y+") in color "+color+"." );
    }
}

File: Random_Files/TypescriptRandom/TR1/Monitor.ts
import {Resolution} from './resolution'
export class Monitor{
    private $model: string;
    private $manufacturer: string;
    private $size: number;
    private $nativeResolution: Resolution;

	constructor($$model: string, $$manufacturer: string, $$size: number, $$nativeResolution: Resolution) {
		this.$model = $$model;
		this.$manufacturer = $$manufacturer;
		this.$size = $$size;
		this.$nativeResolution = $$nativeResolution;
	}
    

    /**
     * Getter model
     * @return {string}
     */
	public get model(): string {
		return this.$model;
    }

    /**
     * Getter manufacturer
     * @return {string}
     */
	public get manufacturer(): string {
		return this.$manufacturer;
	}

    /**
     * Getter nativeResolution
     * @return {Resolution}
     */
	public get nativeResolution(): Resolution {
		return this.$nativeResolution;
	}

    /**
     * Getter size
     * @return {number}
     */
	public get size(): number {
		return this.$size;
	}
    
    
}